src/commands/release/import.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import { createReadStream } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an application or fleet.
+
+		To save a release into a file, use 'balena release export'.
```

Comment:
nit
```suggestion
		To save a release to a file, use 'balena release export'.
```

Personally I also would have gone with `export` instead of `save`, but I don't know if this was discussed anywhere

Reply:
Descriptions updated.